tests/regressions/python/466_compiler_problems.py

```diff
@@ -6,7 +6,9 @@
 # #466: ast compiler hangs and gives the wrong answer
 
 import numpy as np
-from phylanx import Phylanx
+from phylanx import Phylanx, PhylanxSession
+
+PhylanxSession(1)
```

Comment:
Could we make it so that this does not have to be called, in which case some defaults are being used.